File: Problem268.java
import java.util.*;

class Problem268 {
    public int missingNumber(int[] nums) {
        List<Integer> list = new ArrayList<>();
        for (int i = 0; i <= nums.length; i++) list.add(i);
        for (int i: nums) list.remove((Integer)i);
        return list.get(0);
    }
}

File: Problem268.cpp
#include <vector>
using namespace std;

class Problem268 {
public:
    int missingNumber(vector<int>& nums) {
        for (int i = 0; i <= nums.size(); i++){
            if (find(nums.begin(), nums.end(), i) == nums.end()) return i;
        }
        return NULL;
    }
};